knowledge-base/kb-datetimevalues-to-formattedstrings.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Formatting Display of DateTime Values in Selection Box
+description: "How to convert all DateTime values to list of formatted strings."
+type: how-to
+page_title: How to Display DateTime Values as Formatted Strings in Selection Box for WPF
+slug: kb-datetimevalues-to-formattedstrings
+position: 0
+tags: DateTime, AllowMultipleSelection, MultipleSelectionBoxTemplate
+ticketid: 1622029
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>2023.2.718</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>RadComboBox for WPF</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+How to format `DateTime` values to strings in `SelectionBox`.
```

Comment:
There is no SelectionCombo component. I guess you mean RadComboBox? Or the Selection Box element of the RadComboBox? You can link the Visual Structure for better understanding of what is Selection Box.
https://docs.telerik.com/devtools/wpf/controls/radcombobox/general-information-visual-structure